app/views/Home/ProjectItem/index.tsx

```diff
@@ -128,14 +113,23 @@ function ProjectItem(props: RecentProjectItemProps & PropsFromState) {
     }), []);
 
     const convertedProjectActivity = useMemo(() => (
-        projectActivity?.map(pa => ({
+        projectActivity?.map((pa) => ({
             count: pa.count,
             date: (new Date(pa.date)).getTime(),
         })).sort((a, b) => compareDate(a.date, b.date))
     ), [projectActivity]);
 
-    const canEditProject = projectRoles[role]?.setupPermissions?.modify;
-    const canAddEntries = projectRoles[role]?.entryPermissions?.modify;
+    // const canEditProject = projectRoles[role]?.setupPermissions?.modify;
+    // const canAddEntries = projectRoles[role]?.entryPermissions?.modify;
+    const canEditProject = true;
+    const canAddEntries = true;
```

Comment:
Why set these to true?

Reply:
No data in API right now, @k9845 is working on new queries for this.

Reply:
Let's add a FIXME